aospy/calc.py

```diff
@@ -459,33 +459,14 @@ def _create_input_data_obj(self, var, start_date=False,
         # A config setting or some other user input should specify what method
         # to call to access the files on the filesystem.
         utils.io.dmget(paths)
-        ds_chunks = []
-        vars_to_drop = ['time_bounds', 'nv', 'bnds',
-                        'average_T1', 'average_T2']
-        for file_ in paths:
-            test = xr.open_dataset(file_, decode_cf=False,
-                                   drop_variables=vars_to_drop)
-            # Workaround for years < 1678 causing overflows.
-            if start_date < pd.Timestamp.min:
-                freq = test[TIME_STR].attrs['units'].split('since')[0]
-                # We coerce the data to start at pd.Timestamp.min.year + 1.  By
-                # subtracting from this one less than the original start year
-                # when we define the units, the CF decoding step results in an
-                # array that starts in the year pd.Timestamp.min.year + 1.
-                start_yr = pd.Timestamp.min.year + 2 - self.start_date.year
-                units_str = '{0}since {1}-01-01 00:00:00'.format(freq,
-                                                                 start_yr)
-                test[TIME_STR].attrs['units'] = units_str
-                test[TIME_STR].attrs['calendar'] = 'noleap'
-            # 'climatology_bounds' causes the ValueError b/c 'bnds' removed.
-            if self.dtype_in_time == 'av':
-                try:
-                    test = test.drop('climatology_bounds')
-                except ValueError:
-                    pass
-            test = xr.decode_cf(test)
-            ds_chunks.append(test)
-        ds = xr.concat(ds_chunks, dim=TIME_STR, data_vars='minimal')
+        vars_to_drop = ['time_bounds', 'nv', 'bnds', 'average_T1',
```

Comment:
I think we needed this due to an upstream bug in xarray (pydata/xarray#665), which has since been fixed.  Therefore I think we can remove this ```drop_variables``` part of the load step.